### PR TITLE
Feature/issue 44/View list of organizations

### PR DESCRIPTION
**Pull request by TekuriSaiAkhil**
# [Issue 44](https://github.com/gcivil-nyu-org/wed-fall24-team5/issues/44)

## Description

View list of all organizations of a user.
Added authentication check on the landing page.

## Change Type (delete non-relevant options)

- 🪲 Bug fix (non-breaking change which fixes an issue)
- 💡 New feature (non-breaking change which adds functionality)

## Dependencies

- None

### CODE REVIEW

**Review by QahtanAlJammali — APPROVED** [review]
Looking good! thanks sir.

File: src/templates/partials/theader.html [hunk 1]
```diff
@@ -46,7 +46,7 @@
                     <a class="navbar-link">View As</a>
                     <div class="navbar-dropdown">
                         <a href="#" class="navbar-item">User</a>
-                        <a href="#" class="navbar-item">Organization 1</a>
+                        <a href="{% url 'donor_dashboard:org_list' %}" class="navbar-item">Organization 1</a>
```
Comment (arsalananwar11) [inline src/templates/partials/theader.html:49]: Can we change "Organization 1" to just "Organisation"?

**Review by arsalananwar11 — CHANGES_REQUESTED** [review]

File: src/templates/partials/theader.html (hunk 1 above)
Comment (maxgrove42) [reply src/templates/partials/theader.html:49]: I can handle this on front-end as well

**Comment by maxgrove42** [comment]
woohoo ! great work!

**Review by maxgrove42 — APPROVED** [review]

File: src/templates/partials/theader.html (hunk 1 above)
Comment (arsalananwar11) [reply src/templates/partials/theader.html:49]: Yes, sounds good!

**Review by arsalananwar11 — APPROVED** [review]
Looks good, thanks Akhil!

File: src/templates/partials/theader.html (hunk 1 above)
Comment (TekuriSaiAkhil) [reply src/templates/partials/theader.html]: I have updated this part as well. Please take a look.

**Pull request merged**